src/audio/drc/Testbench_drc_lin2dB/norm.h

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
file names and dirs are all lower case

Reply:
OK - I will rename case letter to lower case